netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -197,17 +197,6 @@ public NettyChannelBuilder flowControlWindow(int flowControlWindow) {
   }
 
   /**
-   * Sets the max message size.
-   *
-   * @deprecated Use {@link #maxInboundMessageSize} instead
-   */
-  @Deprecated
-  public NettyChannelBuilder maxMessageSize(int maxMessageSize) {
```

Comment:
We still use this method internally, and while it can be removed per the compat guidelines, we wouldn't be able to sync gRPC internally.  We find a fair number of issues when doing the sync, so removing this now would cause more issues than it fixes.  

I am undertaking the fixing of users, but it takes time.